skyportal/handlers/api/comment.py

```diff
@@ -159,3 +162,35 @@ def delete(self, comment_id):
         self.push_all(action='skyportal/REFRESH_SOURCE',
                       payload={'source_id': source_id})
         return self.success()
+
+
+class CommentAttachmentHandler(BaseHandler):
+    @auth_or_token
+    def get(self, comment_id):
+        """
+        ---
+        description: Download comment attachment
+        parameters:
+          - in: path
+            name: comment_id
+            required: true
+            schema:
+              type: integer
+        responses:
+          200:
+            content:
+              application:
+                schema:
+                  type: string
+                  format: base64
+                  description: base64-encoded contents of attachment
+        """
+        comment = Comment.query.get(comment_id)
```

Comment:
Shouldn't there be a check here to make sure the user should be able to view the comment?

Reply:
See comment below - we do a permissions check